src/ui/Scheduler.hpp

```diff
@@ -102,6 +105,7 @@ struct Scheduler {
             case gr::lifecycle::State::REQUESTED_PAUSE:
             case gr::lifecycle::State::PAUSED:
                 _thread = std::thread([this]() {
+                    gr::thread_pool::thread::setProcessName("UiScheduler #2 (paused)");
```

Comment:
The numbering of the threads is not clear to me here. Is using the acutal thread-id instead of 2 missing here?

My interpretation is that this thread doesn't get it's own id because it is only a short lived helper thread to trigger the state change. Maybe it would be sufficient to change the `#2` to `(temp)` or something like that.